asset-bundle-converter/Assets/AssetBundleConverter/Editor/CustomGltfImporter.cs

```diff
@@ -85,6 +84,21 @@ public override void OnImportAsset(AssetImportContext ctx)
             }
         }
 
+        protected override void PreProcessGameObjects(GameObject sceneGo)
+        {
+            var meshFilters = sceneGo.GetComponentsInChildren<MeshFilter>();
+
+            foreach (MeshFilter filter in meshFilters)
+            {
+                if (filter.name.Contains("_collider", StringComparison.InvariantCultureIgnoreCase))
+                {
+                    Physics.BakeMesh(filter.sharedMesh.GetInstanceID(), false);
+                    filter.gameObject.AddComponent<MeshCollider>();
+                    filter.GetComponent<MeshRenderer>().enabled = false;
```

Comment:
did you try to remove this component? will it work?

Reply:
or maybe it will work only if you assign `collider.sharedMesh = filter.sharedMesh`

Reply:
Ah removing the component is also a good idea, so we just don't have it, but we might break stuff with the renderer logic 🤔